esp-hal/src/gpio/mod.rs

```diff
@@ -1139,6 +1149,7 @@ impl<'d> Output<'d> {
     /// blink_once(&mut led, &mut delay);
     /// # }
     /// ```
+    // FIXME: when https://github.com/esp-rs/esp-hal/issues/2839 is resolved, add an appropriate `# Error` entry.
```

Comment:
Can we add a note on the linked issue with links to this fixmes so we make sure these are resolved before closing the issue